### PR TITLE
Fix playsound module toggling from playsound admin page

### PR DESCRIPTION
**Pull request by pajlada**
- Fix toggling of playsound module from playsound admin page
- Add changelog entry

Not yet tested

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

### CODE REVIEW

**Comment by TroyKomodo** [comment]
Should toggling playsounds require you to click the save button?

File: static/scripts/admin/playsound.admin.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ $(window).on('load', function() {
         $.ajax({
             url: `/api/v1/modules/toggle/playsound`,
             type: 'POST',
-            data: {
-                new_state: checked ? 1 : 0,
-            },
+            contentType: 'application/json; charset=utf-8',
+            data: JSON.stringify({
+                new_state: checked ? true : false,
```
Comment (RAnders00) [inline static/scripts/admin/playsound.admin.js:12]: `checked ? true : false`? Can’t that be simplified to just `checked`?

**Review by RAnders00 — CHANGES_REQUESTED** [review]

**Comment by pajlada** [comment]
> Should toggling playsounds require you to click the save button?

Not really relevant for this PR

**Comment by pajlada** [comment]
Tested this now and it works as expected :ok_hand:

**Review by RAnders00 — APPROVED** [review]

**Pull request merged**